docs/migration/2_0.md

```diff
@@ -181,7 +181,14 @@ Support for functions dropped:
 - `CartItemListComponent` no longer requires `FormBuilder`, `FeatureConfigService` and  `CartService`. `CartService` was replaced with corresponding methods from `ActiveCartService`. The deprecated method `isSaveForLaterEnabled()` was removed.
 - `CartItemComponent` no longer uses `FeatureConfigService`. The deprecated method `isSaveForLaterEnabled()` was removed.
 - `PaymentFormComponent` requires new parameter now: `UserAddressService`.
-
+- `LoaderState`, `ProcessesLoaderState`, `EntityState`, `EntityLoaderState`, `EntityProcessesLoaderState` were moved under `StateUtils` namespace.
```

Comment:
Could you move it to separate section
like `### State utility changes`.

It is not that automated as constructor changes, as we only add comments for broken code.